ArduCopter/Copter.h

```diff
@@ -679,6 +679,8 @@ class Copter : public AP_Vehicle {
     // lua scripts use this to retrieve EKF failsafe state
     // returns true if the EKF failsafe has triggered
     bool has_ekf_failsafed() const override;
+    bool is_landing() override;
```

Comment:
Looks pretty good.  These could be made const I think.

Reply:
Thank you! I fixed it.